napari/utils/geometry.py

```diff
@@ -99,6 +109,37 @@ def rotation_matrix_from_vectors(vec_1, vec_2):
     return rotation_matrix
 
 
+def rotate_points_on_plane(
+    points: np.ndarray,
+    current_plane_normal: np.ndarray,
+    new_plane_normal: np.ndarray,
+) -> Tuple[np.ndarray, np.ndarray]:
+    """Rotate points that all lie on the same plane to a plane with a new normal vector.
```

Comment:
Can you elaborate this a bit? Is this whole plane getting rotated? Are they being projected onto the rotated plane? What's the difference of this compared to simply applying a rotation matrix to a bunch of non-coplanar points?

Reply:
@brisvag agreed this isn't super clear, the transformation applied on the points is one which aligns the two plane normals - we're renaming

Reply:
naming is hard 😆